### DISCUSSION TITLE
Best way to access 4 different Slingboxes?

### DISCUSSION THREAD

**Discussion by Alimond52** [GerryDazoo/Slinger#610] [General]
Hey all,
When I was abroad in March-April, Slinger and SlingerPlayer worked perfectly. Good job!
I have 4 different Slingboxes. And have previously only used a single Slingbox. However, I have made 4 different Config.ini files for each Slingbox, and it works.
I would now like to use the different Slingboxes more easily so I can change boxes faster.
Can anyone tell me which way is the best?
I guess I have to have a Config.ini that contains settings for all 4 boxes? Is there any instruction how to do with all this?
After blocked IP addresses in Router, I haven't had any problems with any boxes being bricked again. Great!
I wish you a nice summer

**Comment by GerryDazoo** [comment]
Use the unified-config.ini format. There's a sample in the repository and documentation in the relase notes.

**Reply by Alimond52** [reply]
Thanks for the help Gerry,
I had made a Unified config.ini file in March with my 4 Slingboxes, but couldn't get it to work then.
After many tests I finally got access to the server.
The result is that the two 500 boxes work well in every way.
But the two ProHD boxes cannot start any current. I just get an error message:
"Error Can't find streamer for" and then the name of the box.
If I run these boxes from normal Config.ini for a single box, it works fine with both ProHDs. I have transferred the settings from these two config.files to the Unified file. So then I think it should work there too.
I have checked the IP addresses, ports, and SB box name. Run from Component source selected Videosource 3 and selected Resolution 15.
Do you have any idea?
English is not my native language, so write clearly please.
Grateful for help!

**Reply by GerryDazoo** [reply]
post the contents of your config.ini and the URLs you are using to try to connect to them with. I'm sure it's something easy to fix.
If you use SlingerPlayer (https://github.com/GerryDazoo/SlingerPlayer)  it queries the Server for the valid URLs and presents you with a list to pick from.

**Comment by Alimond52** [comment]
Hi Gerry, Thanks! It was solved when I read through the Config file carefully, and found an extra bracket = [[ at the SB name, and then the other ProHD, had an SB name with spaces. I removed it and now all 4 boxes worked great! Wonderfully!
